### PR TITLE
More details about SimSwap in the API documentation

### PR DESCRIPTION
**Pull request by gregory1g**
More explicit definition of SimSwap in the API description.

More detailed description of scenarios where SimSwap happens and  API can be used.

#### What type of PR is this?
documentation

#### Which issue(s) this PR fixes:

- Clarify valid SIM Swap scenarios #73 
- Activation date event considered as sim swap event #78 
- Call Center Use Case for SIM Swap #81

### CODE REVIEW

File: code/API_definitions/sim_swap.yaml [hunk 1]
```diff
@@ -6,21 +6,25 @@ info:
     
     # Introduction
 
-    The SIM Swap API performs real-time checks on the activation date of a SIM card on the mobile network. It reveals if an individual mobile phone number (MSISDN) has been ported to another SIM card.
+    The SIM Swap API performs real-time checks on the recent SIM Swap event.
```
Comment (fernandopradocabrillo) [inline code/API_definitions/sim_swap.yaml:9]:
```suggestion
    The SIM Swap API performs real-time checks on the the last SIM Swap event.
```

File: code/API_definitions/sim_swap.yaml [hunk 2]
```diff
@@ -6,21 +6,25 @@ info:
...
+    Depending on the network provider implementation, legal enforcement, etc... either one or both endpoints could be implemented.
 
     # Relevant terms and definitions
 
-    * **SIM swap**:
-    A SIM swap is a process in which a mobile phone user's current SIM card is deactivated or replaced with a new one. This is typically done by contacting the user's mobile service provider and requesting a new SIM card for various reasons, such as a lost or damaged SIM card, upgrading to a new phone, or changing phone numbers while keeping the same device.
+    **SIM swap**:
+    A SIM swap is a process in which a user's mobile phone number (MSISDN) is associated with a new SIM card (IMSI). This is typically done by contacting the user's mobile service provider and requesting a new SIM card for various reasons, such as a lost or damaged SIM card or upgrading to a new phone.
+    
+    SimSwap also happens during other actions like changing user's phone number or changing mobile service provider keeping user's mobile phone number.
```
Comment (fernandopradocabrillo) [inline code/API_definitions/sim_swap.yaml:27]:
```suggestion
    SimSwap also happens during other actions like changing user's phone number, changing mobile service provider keeping user's mobile phone number or when activating a new SIM associated to the same phone number, known as multisim service.
```

**Review by fernandopradocabrillo — COMMENTED** [review]
Thanks @gregory1g for the PR.

Just a couple suggestions, the rest is fine for me

**Review by bigludo7 — APPROVED** [review]
LGTM

**Comment by gregory1g** [comment]
Updated following @fernandopradocabrillo suggestions.

**Review by bigludo7 — APPROVED** [review]

**Review by fernandopradocabrillo — APPROVED** [review]
LGTM

**Comment by fernandopradocabrillo** [comment]
@gregory1g merged since it has 2 codeowner approvals

**Pull request merged**